src/aria/templates/ClassWriter.js

```diff
@@ -495,6 +507,15 @@ module.exports = Aria.classDefinition({
             if (this._curblock) {
                 this.writeln("this['" + Aria.FRAMEWORK_PREFIX + "currentLineNumber'] = " + lineNumber + ";");
             }
+        },
+
+        /**
+         * Set the parseOnly flag to true and disables functions which are only useful for class generation.
+         */
+        enableParseOnly : function () {
+            this.parseOnly = true;
+            // disables functions which are only useful to generate code:
+            this.writeDependencies = this.newVarName = this.writeln = this.write = this.wrapExpression = this.trackLine = Aria.empty;
```

Comment:
I would rather split this line (even though it duplicates code): 

```javascript
this.writeDependencies = Aria.empty;
this.newVarName = Aria.empty;
this.writeln = Aria.empty;
this.write = Aria.empty;
this.wrapExpression = Aria.empty;
this.trackLine = Aria.empty;
```

With nowadays editors features, it's easy to add/remove a method (duplicate/delete line), and visually it's perfectly aligned so that we can know more easily the set of overridden methods at a glance.